### PR TITLE
DOM: Test that some event listeners are passive by default

### PR DESCRIPTION
**Pull request by zcorpan**
Follows https://github.com/whatwg/dom/pull/1087

### CODE REVIEW

**Comment by zcorpan** [comment]
Test results: https://wpt.fyi/results/dom/events/passive-by-default.html?sha=e09aa79d2c&label=pr_head

All tests pass except the `mousewheel` event tests in Firefox because [Firefox doesn't support that event.](https://bugzilla.mozilla.org/show_bug.cgi?id=1529953)

File: dom/events/passive-by-default.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  function isListenerPassiveByDefault(eventName, eventTarget, expectPassive) {
+    test(() => {
+      let defaultPrevented = null;
+      eventTarget.addEventListener(eventName, event => {
+        event.preventDefault();
+        defaultPrevented = event.defaultPrevented;
+      });
+      let dispatchEventReturnValue = eventTarget.dispatchEvent(new Event(eventName, {cancelable: true}));
+      assert_equals(defaultPrevented, !expectPassive, 'defaultPrevented');
+      assert_equals(dispatchEventReturnValue, expectPassive, 'dispatchEventReturnValue');
```
Comment (domenic) [inline dom/events/passive-by-default.html:18]:
```suggestion
      assert_equals(dispatchEventReturnValue, expectPassive, 'dispatchEvent() return value');
```

(optional)

**Review by domenic — APPROVED** [review]
This LGTM. However I wonder if we should also test that passing explicit `{ passive: false }` overrides this default?

**Comment by zcorpan** [comment]
@domenic done, also tested `{passive: undefined}` and `{passive: true}`

**Review by domenic — APPROVED** [review]
Beautiful

**Pull request merged**